src/acceptance/java/io/codurance/pouch/GetResourceFeature.java

```diff
@@ -39,26 +42,26 @@ public void setUp() {
     @Test
     public void shouldGetOneResource() {
         var resourceOne = new DatabaseHelper.Resource(
-                1,
-                "2018-12-05 16:01:00.0",
+                UUID.fromString("0b6f4e90-1295-11e9-ab14-d663bd873d93"),
```

Comment:
Consider generating the UUID with `UUID.randomUUID()` and storing at as a variable above.